src/App.tsx

```diff
@@ -1,61 +1,142 @@
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+const getTodos: Todo[] = todosFromServer.map((todo) => ({
+  ...todo,
+  user: usersFromServer.find((user) => user.id === todo.userId) || null,
+}));
+
+const getTodo = (
+  selectedUser: User,
+  title: string,
+  todos: Todo[],
+): Todo => {
+  const lastId = Math.max(...todos.map(todo => todo.id));
+
+  return ({
+    id: lastId + 1,
+    title,
+    completed: false,
+    userId: selectedUser.id,
+    user: selectedUser,
+  });
+};
+
+export const App: React.FC = () => {
+  const [todos, setTodos] = useState(getTodos);
+  const [userError, setUserError] = useState(false);
+  const [titleError, setTitleError] = useState(false);
+  const [title, setTitle] = useState('');
+  const [userSelected, setUserSelected] = useState('0');
```

Comment:
why string?
Let's make it number if you use user.id for make it an empty string if you use user.name